locales/notes.ca.properties

```diff
@@ -0,0 +1,80 @@
+# General note/notebook operations
+notebooks=Blocs de Notes
+tap-anywhere-to-start=Premeu a qualsevol lloc per iniciar
+note-info=Informacio de la nota
+add-to-contacts=Afegir a Contactes
+new-notebook=Crear Bloc de Notes
+notebook-all=Totes les notes
+notebook-trash=Paperera
+notebook-action-title=Editar Bloc de notes
+notebook-action-rename=Canviar el nom
+notebook-action-delete=Eliminar
+prompt-rename-notebook=Canviar el nom del Bloc de notes:
+prompt-delete-notebook=Premeu OK per eliminar aquest Bloc de notes. \n\nOBSERVACI&Oacute;: totes les vostres notes seran enviades a la paperera.
```

Comment:
@caos30 ,
Are you ok with updating this pull to put the "Paràmetres" and "Observació" strings in the file directly without using the htmlentites? Because I had an update ready to go out, I shipped catalan in version 2.3.0 with those two values swapped in to your translation. So I'd like to get this merged in quickly to match what's currently available in the marketplace.